### PR TITLE
Change license to apache 2.0

### PR DESCRIPTION
**Pull request by luxu1-ms**
license change: MIT License --> Apache License 2.0

### CODE REVIEW

**Comment by shivsood** [comment]
@rajmera3 @arvindshmicrosoft  for review.

**Review by shivsood — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
     </organization>
     <licenses>
         <license>
-            <name>MIT License</name>
+            <name>Apache License 2.0</name>
             <url>http://www.opensource.org/licenses/mit-license.php</url>
```
Comment (rajmera3) [inline pom.xml:17]: @luxu1-ms / @shivsood please update the URL as well:
http://www.apache.org/licenses/LICENSE-2.0.txt

Other than that perfect thanks!

**Review by rajmera3 — CHANGES_REQUESTED** [review]
@

File: pom.xml (hunk 1 above)
Comment (luxu1-ms) [reply pom.xml]: Thank you @rajmera3 Done

**Review by rajmera3 — APPROVED** [review]

**Review by rajmera3 — APPROVED** [review]

**Review by shivsood — APPROVED** [review]

**Pull request merged**